tf2onnx/tfonnx.py

```diff
@@ -1255,6 +1255,45 @@ def onehot_op(ctx, node, name, args):
         return
 
 
+def onehot_op9(ctx, node, name, args):
+    # T output = OneHot(uint8/int32/int64 input, T depth, T on-value, T off-value, @int axis, @dtype)
```

Comment:
let's follow onnx spec by default, assume all types are supported.
for now, we can make use of target rs6 for onnxruntime as a workaround...

Reply:
done